### PR TITLE
Update the BALANC3R script

### PR DESCRIPTION
**Pull request by gregcowell**
This fixes issue #8. I've submitted a separate pull request to update the GyroBalancer class in the core library.

### CODE REVIEW

**Comment by gregcowell** [comment]
**balanc3r.py**: updated script which relies on GyroBalancer class in core library

**segway.py, parameters.py**: Laurens Valk's scripts with some updates to get them working with the latest core library (can remove these if we do not want to include them)

File: robots/BALANC3R/segway.py [hunk 1]
```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python3
```
Comment (dwalton76) [inline robots/BALANC3R/segway.py]: This file looks like a duplicate of GyroBalancer.py?

Comment (dwalton76) [reply robots/BALANC3R/segway.py]: If we chop this we can also chop parameters.py....but maybe I am missing something on why we needs these

Comment (gregcowell) [reply robots/BALANC3R/segway.py]: I've removed both segway.py and parameters.py.

**Review by dwalton76 — APPROVED** [review]

**Comment by dwalton76** [comment]
@dlech looks like I lost write access to the repo I can no merge PRs

**Comment by dlech** [comment]
Should be fixed now, try it again.

**Comment by dwalton76** [comment]
Yep that did it

**Pull request merged**